composer.json

```diff
@@ -31,7 +31,8 @@
         "skilldisplay/phptoolkit": "^1.2",
         "typo3/cms-backend": "^10.4",
         "typo3/cms-frontend": "^10.4",
-        "typo3fluid/fluid": "^2.6"
+        "typo3fluid/fluid": "^2.6",
+        "guzzlehttp/guzzle": "^6.5"
```

Comment:
what's the reason to specifically require guzzle here? The core already does require `^6.3` doesn't that suffice?

Reply:
The SDK requires Guzzle and uses type hints against Guzzle, instead of PSR. We now also explicitly use Guzzle within this extension. Therefore we state the dependency of our Extension.
We also have a dependency check within GitHub Actions that ensures all used APIs are available. Without adding the dependency, that would fail, because we don't state the dependency.

That's already documented within the commit message, let me add the quote:

>     Since we now interact with guzzle, we add the dependency.
>     Wouldn't be necessary if SDK would have PSR compatible type hints.

It would be cool to just use PSR here, but that would require the SDK to switch to PSR as well, which is out of scope for this PR.

Reply:
Alright, thanks for the explanation. I honestly didn't read all commit messages, I only looked at the code. Sorry for that.